e2e/wdio/core/tests/form-message.e2e-spec.ts

```diff
@@ -0,0 +1,143 @@
+    const { messageWithInput, messageWithInputGroup, buttons, messageInformation, messageWithTextArea, inputFields } = formMessagePage;
+
+    beforeAll(() => {
+        formMessagePage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(messageWithInput);
+    }, 1);
+
+    it('should check message with input field', () => {
+        scrollIntoView(messageWithInput);
+        expect(getAttributeByName(messageWithInput, 'placeholder')).toBe(text);
+        setValue(messageWithInput, testText);
+        expect(getValue(messageWithInput)).toBe(testText);
+        expect(getText(messageInformation)).toBe(hoverEventMessageInput);
+    });
+
+    it('should check message with input group field', () => {
+        scrollIntoView(messageWithInputGroup, 1);
+        setValue(messageWithInputGroup, testText, 1);
+        expect(getValue(messageWithInputGroup, 1)).toBe(testText);
+        click(buttons, 1);
+        expect(getText(messageInformation)).toBe(clickEventMessage);
+    });
+
+    it('should check message with input group field - hover', () => {
+        scrollIntoView(messageWithInputGroup, 2);
+        mouseHoverElement(messageWithInputGroup, 2);
+        expect(getText(messageInformation)).toBe(hoverEventMessageInputGroup);
+        setValue(messageWithInputGroup, testText, 2);
+        expect(getValue(messageWithInputGroup, 2)).toBe(testText);
+        expect(isElementClickable(buttons, 2)).toBe(true);
+    });
+
+    it('should check message with textarea', () => {
+        scrollIntoView(messageWithTextArea);
+        setValue(messageWithTextArea, testMultilineText);
+        expect(getValue(messageWithTextArea)).toBe(testMultilineText);
+        expect(getText(messageInformation)).toBe(eventMessageTextArea);
+    });
+
+    describe('Check visual regression', function() {
+
+        it('should check examples visual regression', () => {
+            formMessagePage.saveExampleBaselineScreenshot();
+            expect(formMessagePage.compareWithBaseline()).toBeLessThan(2);
+        });
+
+        it('verify input fields states', () => {
+            const inputLength = getElementArrayLength(inputFields);
+            for (let i = 0; i < inputLength; i++) {
+                scrollIntoView(inputFields, i);
+                checkElementStates(inputFields, inputField + i + '-', input, i);
+            }
+        });
+
+        it('verify buttons states', () => {
+            const buttonsLength = getElementArrayLength(buttons);
+            for (let i = 1; i < buttonsLength; i++) {
+                scrollIntoView(buttons, i);
+                checkElementStates(buttons, button + i + '-', btn, i);
+            }
+        });
+
+        it('verify message information 1st input field hover state', () => {
+            scrollIntoView(messageWithInput);
+            click(messageWithInput);
+            checkElementHoverState(messageInformation, message + '-0' + 'hover-state-', msg);
```

Comment:
Doesn’t  look like it will have the same screenshot name. 
Your current screenshots says  `-0-hover-state`
Your code says `-0hover-state`

Reply:
yep, checked that your PR generating new screenshots in the run time 
#####################################################################################
 INFO:
 Autosaved the image to
 /home/travis/build/SAP/fundamental-ngx/e2e/wdio/baselineScreenshot/core/form-message/desktop_microsoftedge/form-message-message--0hover-state-msedge-windows--1920x1080.png
#####################################################################################